src/Psl/Xml/Exception/XmlErrorsException.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Xml\Exception;
+
+use LibXMLError;
+use Psl\Exception\RuntimeException;
+use Psl\Xml;
+
+final class XmlErrorsException extends RuntimeException
```

Comment:
we need `Psl\Xml\Exception\ExceptionInterface`, and the name here should probably change 🤔 

`XmlErrorsException` doesn't really give out anything.